lang/cs/Org.Apache.REEF.Common/Runtime/Evaluator/Context/ContextRuntime.cs

```diff
@@ -210,17 +227,21 @@ public ContextRuntime SpawnChildContext(IConfiguration childContextConfiguration
 
                     var e = new InvalidOperationException(message);
                     Utilities.Diagnostics.Exceptions.Throw(e, LOGGER);
+                    throw e;
```

Comment:
I only changed it to let the compiler and VS static code analysis know that an Exception is to be thrown there. I'd be happy to remove `Exceptions.Throw` :).